db/create/create_tables.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE Type (
+  id INT NOT NULL AUTO_INCREMENT,
+  name varchar(45) NOT NULL UNIQUE,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE kanto . TypeEffect(
+id INT NOT NULL AUTO_INCREMENT,
+power DOUBLE NOT NULL,
+description varchar(45),
+PRIMARY KEY(id));
+
+
+CREATE TABLE TypeMatchup (
+  attackingTypeId INT NOT NULL,
+  defendingTypeId INT NOT NULL,
+  typeEffectId INT NOT NULL,
+  PRIMARY KEY (attackingTypeId, defendingTypeId, typeEffectId),
+  FOREIGN KEY (attackingTypeId) REFERENCES kanto.type (id),
+  FOREIGN KEY (defendingTypeId) REFERENCES kanto.type (id),
+  FOREIGN KEY (typeEffectId) REFERENCES kanto.typeeffect (id)
+);
+
+CREATE TABLE Pokemon (
+    id INT NOT NULL AUTO_INCREMENT,
+    nationalDexNumber int NOT NULL,
+    name varchar(45) NOT NULL,
+    type1Id INT NOT NULL,
+    type2Id INT DEFAULT NULL,
```

Comment:
Double check if  `DEFAULT NULL` is necessary here. All fields are nullable by default if I am not mistaken.